testing/framework/TestCommon.py

```diff
@@ -268,6 +268,15 @@ def must_be_writable(self, *files):
     def must_contain(self, file, required, mode = 'rb', find = None):
         """Ensures that the specified file contains the required text.
```

Comment:
can you add a docstring for this method?

Reply:
It has a docstring... and the description of the method doesn't change, we're still looking for a bit of text in a file.